src/Model/IssueModel.php

```diff
@@ -5,6 +5,7 @@
 namespace Icanhazstring\RandomIssuePicker\Model;
 
 use JMS\Serializer\Annotation as Serializer;
+use Datetime;
```

Comment:
Using `DateTime` inherits the problem of a mutable state of this object.
Please use `DateTimeImmutable` for this purpose.